src/sparseml/pytorch/optim/modifier_quantization.py

```diff
@@ -95,6 +97,8 @@ class QuantizationModifier(ScheduledModifier):
     :param reduce_range: if True, the quantization range will be reduced by one bit.
         This may prevent overflow issues with model execution on certain hardware
         Default is False
+    :param activation_bits: Number of bits to use for setting quant min/max values for
```

Comment:
nit: should go below `quantize_linear_activations`

Reply:
Done